ph5/utilities/utmtolatlong.py

```diff
@@ -0,0 +1,167 @@
+# This script can be used to convert UTM coordinates to Latitude/Longitude
+# Batch method: use input file like utmdata.txt:
+# top line is header, subsequent lines are Easting, Northing, Zone, Hemisphere
+#     easting, northing, zone, hemisphere
+#     540947,1361594,58,S
+#     322916,3771967,13,N
+
+# utmtolatlong -i utmdata.txt -o mylatlong.txt
+
+# Interactive method:
+#    enter -e Easting, -n Northing, -z Zone (1-60), -s Hemisphere (S, N)
+# example:
+# utmtolatlong -e 540947 -n 1361594 -z 58 -s S
+# which results in output:
+# easting=540947.0, northing=1361594.0, zone=58, hemisphere=S
+# Lon = 166.7381664, Lat = -77.8156740
+
+# Batch and Interactive methods are exclusive, use one or the other.
+
+# Dave Thomas, 2019-07-31
+
+from __future__ import (print_function)
+from pyproj import Proj, transform
+import argparse
+
+PROG_VERSION = '2019.212'
+
+
+class utm_conversions:

```

Comment:
Class names should always be upper case. I suggest changing this class name to `UTMConversions`